client/app/hosting/ftp/FTP.html

```diff
@@ -1,54 +1,70 @@
 <div class="container-fluid px-0" data-ng-controller="HostingTabFTPCtrl as ctrlHostingTabFTP">
+    <div class="loader mt-5 text-center" aria-label="{{i18n.common_loading}}" data-ng-if="ctrlHostingTabFTP.loading.init"></div>
```

Comment:
Why the loader has changed ? 
@rfougier change them to `<oui-loader></oui-loader>` recently. 

It can even be optimized by `<oui-loader size="l" data-ng-if="ctrlHostingTabFTP.loading.init"></oui-loader>` :)